### ISSUE TITLE
inline delete impossible

### ISSUE THREAD

**Issue by bouks** [openpsa/grid.js#79]
Bug in the confirmation modal (no button to validate or cancel)

Must merge before to see and fix this bug : https://github.com/bouks/grid.js/commit/9d3cbf427487e27220fab91a6d5ad1957db3f89b#diff-382b3649283911e1a5c64f3ec00e4968R1595

**Comment by flack** [comment]
Can this be closed? The referenced commit is merged

**Comment by bouks** [comment]
THis one, no, it's an actual bug on the modal.

"Still impossible to delete -> now bug in the confirmation modal (no button to validate or cancel)"

**Comment by flack** [comment]
Ah, ok. I must have overlooked the "modal" part :-)

**Comment by bouks** [comment]
I think the problem might be in all the delete (inline or not).
Oleg made much modifications to modals. The problem certainly comes from his commits or meh-uk "imports".
It should be fixed quickly, with no possible deletion, the grid is almost non-usable. @OlegKi @meh-uk you know this part of code, could you look at this ?

**Comment by OlegKi** [comment]
@bouks Thanks! It was my bug in [the changes](https://github.com/OlegKi/jqGrid/commit/60a5496851ab986a784fd098d97952c035fbf559). It's [fixed](https://github.com/OlegKi/jqGrid/commit/7abcfb3117d9df20e3db2e536d86da1b5e78e8b4) now.

**Comment by bouks** [comment]
We already have this [fixed](https://github.com/OlegKi/jqGrid/commit/7abcfb3117d9df20e3db2e536d86da1b5e78e8b4) change [here](https://github.com/openpsa/grid.js/blob/master/external/jquery.fmatter.js#L241)

Still buggy. No "cancel" or "ok" button in the modal.

**Comment by OlegKi** [comment]
@bouks  Do you tried some demos which uses my code? For example try [the demo](http://www.ok-soft-gmbh.com/jqGrid/OK/navButtons0.htm) or [this one](http://www.ok-soft-gmbh.com/jqGrid/OK/navButtons0-fa4.htm). I don't see any problems.

**Comment by bouks** [comment]
@OlegKi In your example the modal is too big, this is the same bug.

Fixed here.
https://github.com/openpsa/grid.js/commit/e0f44aaf4a0d4e93c38eebe14ccc1874a0c8f825

**Comment by flack** [comment]
@bouks could you add some before & after screenshots? I've clicked the links, but it is not obvious to me what you mean? You can just drag and drop image files into the comment boxes here on github

**Comment by bouks** [comment]
![too-little-before](https://cloud.githubusercontent.com/assets/2734718/5900374/97adb78c-a568-11e4-8c44-20ea15276bf8.jpg)
![too-little-after](https://cloud.githubusercontent.com/assets/2734718/5900377/9b41384c-a568-11e4-9858-f5e716968e01.jpg)

**Comment by flack** [comment]
Thanks! Now I get. After really looks much better.

Do you think it would also be possible to center the dialog relative to the grid?

**Comment by bouks** [comment]
Why wouldn't it be possible ? :)

I have no time for this right now. I just wanted to fix a critic bug.

So if no objection about this fix, i propose to close this issue (and then this one too https://github.com/bouks/grid.js/commit/9d3cbf427487e27220fab91a6d5ad1957db3f89b#diff-382b3649283911e1a5c64f3ec00e4968R1595) that is a critic bug and open a new one only for centering.

**Comment by flack** [comment]
the delete fix has already been merged here:

https://github.com/openpsa/grid.js/commit/9d3cbf427487e27220fab91a6d5ad1957db3f89b#diff-382b3649283911e1a5c64f3ec00e4968R1595

Your other commits are also already on master as far as I can see it. I don't have access to my dev machine right now, so I'm just going to trust you that they work :-)

If you have some sample code for testing this, it would be nice if you could add it as a new demo (especially since we have nothing showing the editing modes yet).

For future changes, it would be nice if you submit them as pull requests first, because that makes it easier to review. Stuff that changes documentation, build system or other auxiliary parts can be committed directly, but IMHO we should be extra careful when changing the actual JS code.

**Comment by flack** [comment]
Tested and verified fixed. I've added a simple inlineedit demo and filed a follow-up ticket for the popup rendering
